shell/common/shorebird/shorebird.cc

```diff
@@ -79,6 +80,113 @@ FileCallbacks ShorebirdFileCallbacks() {
   };
 }
 
+// FIXME: consolidate this with the other ConfigureShorebird
+bool ConfigureShorebird(const ShorebirdConfigArgs& args,
+                        std::string* patch_path) {
+  // FIXME: This was commented out because the windows flutter engine does not
+  //     populate the settings snapshots. Ideally we would call
+  //     ConfigureShorebird from the embedder and this would be uncommented.
+  // If you are crashing here, you probably are running Shorebird in a Debug
```

Comment:
Yeah, this was just a hacky ASSERT used to catch a common debugging mistake when using a local engine for iOS.  We can remove it and/or move it into the iOS platform caller.

Reply:
Removed